packages/sirius-web/backend/sirius-web-services/src/main/java/org/eclipse/sirius/web/services/editingcontext/EditingDomainFactoryService.java

```diff
@@ -56,16 +65,23 @@ public AdapterFactoryEditingDomain createEditingDomain(String editingContextId)
         AdapterFactoryEditingDomain editingDomain = new AdapterFactoryEditingDomain(this.composedAdapterFactory, new BasicCommandStack());
         ResourceSet resourceSet = editingDomain.getResourceSet();
 
+        var isStudioProjectNature = this.editingContextMetadataProvider.getMetadata(editingContextId).natures().stream().map(Nature::natureId)
+                .anyMatch("siriusComponents://nature?kind=studio"::equals);
+
         EPackageRegistryImpl ePackageRegistry = new EPackageRegistryImpl();
-        this.globalEPackageRegistry.forEach(ePackageRegistry::put);
         List<EPackage> additionalEPackages = this.editingContextEPackageService.getEPackages(editingContextId);
-        additionalEPackages.forEach(ePackage -> ePackageRegistry.put(ePackage.getNsURI(), ePackage));
+        Stream.concat(findGlobalEPackages(), additionalEPackages.stream())
+                .filter(ePackage -> !isStudioProjectNature || STUDIO_PROJECT_NATURE_EPACKAGE_NS_URI_AVAILABLE.contains(ePackage.getNsURI()))
+                .forEach(ePackage -> ePackageRegistry.put(ePackage.getNsURI(), ePackage));
```

Comment:
Is it normal I am able to create a view or a domain model on a project that does not have the studio nature?